code/modules/projectiles/guns/projectile/boltgun.dm

```diff
@@ -234,7 +234,7 @@
 	icon_state = "starstriker"
 	item_state = "starstriker"
 	max_shells = 10
-	price_tag = 500 //no...
+	price_tag = 250 //no...
```

Comment:
This is already heavily lowered from a normal guns price do to factorial memes

Reply:
Every gun got reduced by 50% so this brings it in line with all other firearms.